packages/nimble-components/docs/component-status.mdx

```diff
@@ -1,5 +1,8 @@
 import { Story, Meta } from '@storybook/addon-docs';
-import { componentStatus } from './component-status.stories';
+import {
+    componentStatus,
+    componentStatusFuture
+} from './component-status.stories';
 
 <Meta title="Component Status" />
 
```

Comment:
nit: I find myself wanting a sub-heading above the first table now, to balance the "Future work" subheading over the second table. Haven't thought of a great title, but something like:
- `## Available Components`
- `## Complete or In Progress`
- `## Ready and Incubating`

I can also live with it as-is.

Reply:
Added a L2 heading "Ready or incubating" and made legend an L3 heading